### PR TITLE
feat(code): Workday/Netsuite integration - First commit 

### PR DESCRIPTION
**Pull request by JCMOSCON1976**
Description
Workday/Netsuite

JIRA TICKET ASP [ASP-5206](https://mozilla-hub.atlassian.net/browse/ASP-5206)
JIRA EPIC TICKET [ASP-5165](https://mozilla-hub.atlassian.net/browse/ASP-4509)

### CODE REVIEW

File: jobs/eam-integrations/scripts/workday_netsuite_integration.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
+from argparse import ArgumentParser
+import logging
+from workday_netsuite.api.workday import WorkDayRaaService
+from api.util import Util
+
+ 
+class WorkdayToNetsuiteIntegration():
```
Comment (fbertsch) [inline jobs/eam-integrations/scripts/workday_netsuite_integration.py:7]: I have absolutely 0 context on what you're doing here, but I wanted to make sure you've [seen this](https://docs.getcensus.com/destinations/available-destinations/netsuite#supported-objects), which may be an easier way to upload data to Netsuite.

File: jobs/eam-integrations/scripts/workday_netsuite/api/workday/workday.py [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+class WorkDayRaaService():
+    """Workday RaaS service implementation""" 
+    def __init__(self):
+        self.config = config
+        self.logger = logging.getLogger(self.__class__.__name__)
+        
+    def get_listing_of_workers(self) -> list[Worker]:
+
+        """Get  listing of workers report data from WorkDay"""
```
Comment (fbertsch) [inline jobs/eam-integrations/scripts/workday_netsuite/api/workday/workday.py:54]: Just checking that you looked into using Fivetran to import this data?

**Review by fbertsch — APPROVED** [review]
Looks mostly fine from my end, but I'd recommend adding some tests to verify behavior

File: jobs/eam-integrations/scripts/workday_netsuite/api/workday/workday.py (hunk 2 above)
Comment (sean-rose) [reply jobs/eam-integrations/scripts/workday_netsuite/api/workday/workday.py:54]: HR has both [Workday HCM](https://fivetran.com/docs/connectors/applications/workday-hcm) and [Workday RaaS](https://fivetran.com/docs/connectors/applications/workday-raas) connectors set up in Fivetran.

**Pull request merged**